android-configure

```diff
@@ -16,5 +16,5 @@ export PLATFORM=android
 
 if [ $2 -a $2 == 'gyp' ]
   then
-    ./gyp_uv.py -Dtarget_arch=arm -DOS=android
+    ./gyp_uv.py -Dtarget_arch=arm -DOS=android -f make-android
```

Comment:
By target I mean the -f make-android. We never had it in our script, is it
really necessary?
On Jul 10, 2015 4:03 AM, "Ground Up Works" <notifications@github.com> wrote:

> In android-configure
> <https://github.com/libuv/libuv/pull/433#discussion_r34322957>:
>
> > @@ -16,5 +16,5 @@ export PLATFORM=android
> >
> >  if [ $2 -a $2 == 'gyp' ]
> >    then
> > -    ./gyp_uv.py -Dtarget_arch=arm -DOS=android
> > +    ./gyp_uv.py -Dtarget_arch=arm -DOS=android -f make-android
>
> If we don't specify the target as "android", the gyp scripts don't handle
> android cross compilation properly and end up trying to compile against a
> darwin target environment and using the wrong include files, libs, etc.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/libuv/libuv/pull/433/files#r34322957>.
>